code/game/objects/items/ego_weapons/waw.dm

```diff
@@ -870,8 +870,8 @@
 	attack_speed = 1.2
 	damtype = RED_DAMAGE
 	armortype = RED_DAMAGE
-	attack_verb_continuous = list("pokes", "jabs", "tears", "lacerates", "gores")
-	attack_verb_simple = list("poke", "jab", "tear", "lacerate", "gore")
+	attack_verb_continuous = list("pokes", "jabs", "tears", "lacerates", "gores", "gazes upon")
```

Comment:
This is bad, just take it out.

Reply:
No need to be a dick about it, man. Yeesh.